### PR TITLE
TLC-267 create footer.js spec file

### PR DESCRIPTION
**Pull request by cyxoi1**

### CODE REVIEW

File: pageObjects/footer.page.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    this.termsOfService.click();
+  }
+
+  navToPrivacyPolicy() {
+    this.privacyPolicy.click();
+  }
+
+  //this is to validate navigation
+
+  get contactUsPageHeader() {
```
Comment (vasylynash) [inline pageObjects/footer.page.js:34]: No change from the previous PL. See Julia's comments for previous version

Comment (cyxoi1) [reply pageObjects/footer.page.js:34]: I changed the file. I don't know why the old version is displayed here

Comment (cyxoi1) [reply pageObjects/footer.page.js:34]: https://github.com/rinsal/QA6_Local_Coding/pull/42

File: 20608/specs/smoke/footer.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+describe('FOOTER FUNCTIONAL', () => {
+  before(() => {
+    FooterPage.open();
+    browser.maximizeWindow();
+  });
+
+  it('check footer text ', () => {
+    expect(FooterPage.footer.getText()).eq(
+      'If you have any questions or concerns, please contact us.\n' + 'Terms of Service Privacy Policy',
```
Comment (jstepiko) [inline 20608/specs/smoke/footer.js:15]: Refactor of the footer is needed (from the PO)
hardcoded expected result on the line 15

**Pull request merged**